_maps/map_files/cyberiad/z3.dmm

```diff
@@ -1,357 +1,234 @@
+"az" = (/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"},/obj/structure/cable{d1 = 1; d2 = 4; icon_state = "1-4"; tag = ""},/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
+"aA" = (/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"; tag = ""},/obj/structure/cable{d1 = 1; d2 = 8; icon_state = "1-8"; tag = ""},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0; tag = ""},/turf/simulated/floor/plating/airless/catwalk,/area/space)
+"aB" = (/obj/structure/cable{d1 = 1; d2 = 8; icon_state = "1-8"; tag = ""},/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"; tag = ""},/turf/simulated/floor/plating/airless/catwalk,/area/space)
+"aC" = (/turf/simulated/floor/plasteel{tag = "icon-vault (NORTH)"; name = "Mainframe floor"; icon_state = "whitebluefull"; dir = 1; oxygen = 0; nitrogen = 100; temperature = 80},/area/tcommsat/server)
+"aD" = (/turf/simulated/floor/plasteel{dir = 1; icon_state = "darkpurplefull"; name = "Mainframe floor"; nitrogen = 100; oxygen = 0; tag = "icon-vault (NORTH)"; temperature = 80},/area/tcommsat/server)
+"aE" = (/obj/machinery/telecomms/broadcaster/preset_left,/turf/simulated/floor/bluegrid{name = "Mainframe Base"; nitrogen = 100; oxygen = 0; temperature = 80},/area/tcommsat/server)
+"aF" = (/obj/machinery/telecomms/broadcaster/preset_right,/turf/simulated/floor/bluegrid{name = "Mainframe Base"; nitrogen = 100; oxygen = 0; temperature = 80},/area/tcommsat/server)
+"aG" = (/turf/simulated/floor/plasteel{dir = 1; icon_state = "darkbrownfull"; name = "Mainframe floor"; nitrogen = 100; oxygen = 0; tag = "icon-vault (NORTH)"; temperature = 80},/area/tcommsat/server)
+"aH" = (/turf/simulated/floor/plasteel{tag = "icon-vault (NORTH)"; name = "Mainframe floor"; icon_state = "whitegreenfull"; dir = 1; oxygen = 0; nitrogen = 100; temperature = 80},/area/tcommsat/server)
+"aI" = (/obj/machinery/camera{c_tag = "Telecomms West Solars"; dir = 8; network = list("Telecomms")},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/machinery/light{dir = 4},/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
+"aJ" = (/obj/machinery/telecomms/bus/preset_one,/turf/simulated/floor/plasteel{dir = 8; icon_state = "vault"; name = "Mainframe floor"; nitrogen = 100; oxygen = 0; tag = "icon-vault (NORTH)"; temperature = 80},/area/tcommsat/server)
+"aK" = (/obj/machinery/telecomms/processor/preset_one,/turf/simulated/floor/plasteel{dir = 8; icon_state = "vault"; name = "Mainframe floor"; nitrogen = 100; oxygen = 0; tag = "icon-vault (NORTH)"; temperature = 80},/area/tcommsat/server)
+"aL" = (/obj/machinery/telecomms/hub/preset,/turf/simulated/floor/plasteel{dir = 8; icon_state = "vault"; name = "Mainframe floor"; nitrogen = 100; oxygen = 0; tag = "icon-vault (NORTH)"; temperature = 80},/area/tcommsat/server)
```

Comment:
In addition, do you know what the old sat was REALLY, REALLY missing? A way to cool down the server room.

Reply:
Good poitn